vars/checkPatch.groovy

```diff
@@ -41,11 +61,24 @@ def call(Map config) {
                  context: "pre-build" + "/" + env.STAGE_NAME,
                  status: "PENDING"
 
-    script = 'GH_USER="' + config['user'] + '"' + \
-             ' GH_PASS="' + config['password'] + '"' + \
-             ' CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
-             ' jenkins/code_review/jenkins_github_checkwarn.sh'
-    rc = sh(script: script, returnStatus: true)
+    rc = 1
+    if (config['review_creds']) {
+      withCredentials([[$class: 'UsernamePasswordMultiBinding', 
+                      credentialsId: config['review_creds'],
+                      usernameVariable: 'GH_USER',
+                      passwordVariable: 'GH_PASS']]) {
+        script = 'CHECKPATCH_IGNORED_FILES="' + ignored_files + '"' + \
+                 ' jenkins/code_review/jenkins_github_checkwarn.sh'
```

Comment:
This should be defined outside of this block, up before line 63, perhaps, then it can be used here without modification and can be re-used in lines 76-79 by simply prefixing it with the two additional env. variables.

Reply:
Done on latest patch pushed.